tests/Unit/Geometry/PointTest.php

```diff
@@ -2,17 +2,16 @@
 
 declare(strict_types=1);
 
-namespace GeoJson\Tests\Geometry;
+namespace GeoJson\Tests\Unit\Geometry;
 
 use GeoJson\GeoJson;
 use GeoJson\Geometry\Geometry;
 use GeoJson\Geometry\Point;
-use GeoJson\Tests\BaseGeoJsonTest;
+use GeoJson\Tests\Unit\BaseGeoJsonTest;
 use InvalidArgumentException;
 use stdClass;
-
-use function is_subclass_of;
 use function func_get_args;
+use function is_subclass_of;
```

Comment:
Thanks. Would you mind adding the following rule to `phpcs.xml.dist` so we can catch this in the future?

```xml
<rule ref="SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses"/>
```